assets/airports/essa.json

```diff
@@ -0,0 +1,3558 @@
+            "rwy": {
+                "ESSA19R": [],
+                "ESSA19L": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB", "ERK"]
+            ]
+        },
+        "XILAN5T": {
+            "icao": "XILAN5T",
+            "name": "Xilan Five Tango",
+            "holds": {
+                "XILAN": "069|left|1.5min|S230-",
+                "TEB"  : "178|right|1min|S230-"
+            },
+            "entryPoints": {
+                "XILAN": [["XILAN", "A120-"]]
+            },
+            "body": ["SA570", "@TEB"],
+            "rwy": {
+                "ESSA26": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB"]
+            ]
+        }
+    },
+    "_comment": "Data for spawnpatterns is 2018/2019 (pre-covid) statistics from https://en.wikipedia.org/wiki/Stockholm_Arlanda_Airport",
+    "_comment": "London example:",
+    "_comment": "a. ['Passengers handled 2019' 1420655] / ['Total passenger traffic 2017' 26642034] = ['Passenger proportion to/from London' 0.0533] (using 2017 total because 2019 is not available, should be quite representative)",
+    "_comment": "b. ['Total landings 2017' 122697] * ['Passenger proportion to/from London' 0.0533] = ['Landings from London' 6540] (using 2017 total because 2019 is not available)",
+    "_comment": "c. ['Landings from London' 6540] / ['Hours in a year' 8760] / (2/3) = ['Aircraft per hour from London' 1.1] (2/3 because I assume almost no passenger traffic during the night 1/3 of each day)",
+    "_comment": "d. Divide the 1.1 rate between London/Heathrow and London/Gatwick by feeling. This gives me 0.7 to Heathrow and 0.4 to Gatwick",
+    "_comment": "e. Find which airlines operate these destinations and guess the weights by feeling",
+    "_comment": "f. Find a reasonable route from London to Arlanda on http://flightplandatabase.com and select a reasonable STAR",
+    "_comment": "g. Create a spawnPatterns for arrivals from Heathrow and Gatwick using the above information",
+    "_comment": "h. Find a reasonable SID and create corresponding spawnPatterns for departures (assuming landing rate and takeoff rate are equal)",
+    "spawnPatterns" : [
+        {
+            "origin": "EGLL",
+            "destination": "ESSA",
+            "category": "arrival",
+            "route": "INVOL.N866.BEDLA..ELTOK.ELTOK7M.ESSA01R",
+            "altitude": "18000",
+            "speed": "330",
+            "method": "random",
+            "rate": 0.7,
+            "airlines": [
+                ["sas/short", 1],
+                ["baw/short", 1]
+            ]
+        },
+        {
+            "origin": "EGKK",
+            "destination": "ESSA",
+            "category": "arrival",
+            "route": "INVOL.N866.BEDLA..ELTOK.ELTOK7M.ESSA01R",
+            "altitude": "18000",
+            "speed": "330",
+            "method": "random",
+            "rate": 0.4,
+            "airlines": [
+                ["ezy", 1],
```

Comment:
Do I need to normalize the weights so that they add up to something specific (related to the rate) or is this done automatically? My guess is that only the relative weights are important, but that the absolute numbers don't matter.

Reply:
No, I don't need to normalize the weights.